### PR TITLE
[Admin][Order][Customer] Add KnpMenu for order and customer show pages

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
@GSadee @TheMadeleine Let's add some documentation (to the existing customization guide article with menus, I guess) and we can merge this. ;)

File: docs/customization/menu.rst [hunk 1]
```diff
@@ -238,23 +245,32 @@ type is default to make the example easily customizable.
...
-                ->setLabelAttribute('icon', 'star')
-                ->setLabelAttribute('color', 'maroon')
-            ;
+            $order = $event->getOrder();
+            $stateMachine = $event->getStateMachine();
+
+            if ($stateMachine->can(OrderTransitions::TRANSITION_FULFILL)) {
+                $menu
+                    ->addChild('fulfill', [
+                        'route' => 'sylius_admin_order_fulfill',
```
Comment (TheMadeleine) [inline docs/customization/menu.rst]: As there is no such route in Sylius, you should give a warning about it before the example.

```
.. warning::

    There is no ``sylius_admin_order_fulfill`` route in Sylius. Create this route if you need it. 
```

File: docs/customization/menu.rst [hunk 2]
```diff
@@ -238,23 +245,32 @@ type is default to make the example easily customizable.
```
Comment (TheMadeleine) [inline docs/customization/menu.rst]: Before this example and the above one, we should describe what exactly are we doing.

```
In the example below, we are adding one new button to the Admin Order Show Menu. It is a ``transition`` type button, that will let the admin to fulfill the order.
```

File: docs/customization/menu.rst (hunk 1 above)
Comment (GSadee) [inline docs/customization/menu.rst]: Done 👍

File: docs/customization/menu.rst (hunk 2 above)
Comment (GSadee) [inline docs/customization/menu.rst]: Done 👍

**Comment by pjedrzejewski** [comment]
Nice work Grzesiu, thanks! :)

**Pull request merged**